packages/@react-aria/virtualizer/src/VirtualizerItem.tsx

```diff
@@ -65,7 +67,7 @@ export function layoutInfoToStyle(layoutInfo: LayoutInfo, dir: Direction, parent
     WebkitTransitionDuration: 'inherit',
     transitionDuration: 'inherit',
     width: layoutInfo.rect.width,
-    height: layoutInfo.rect.height,
+    height: layoutInfo.isSticky ? '100%' : layoutInfo.rect.height,
```

Comment:
weird that the checkbox shifts down without this, bit concerned that this would mean the computed layout height is no longer used

Reply:
wonder if this is a content vs border box issue with box-sizing, the size of something could change when it gains the selected or focused border
do you know what the height is changing from/to?

Reply:
The height of the checkbox cell doesn't actually change. I think the shift is due to showing the row focus indicator (which has `height: 100%`). The row has height `41`, and the checkbox cell has height `40`.

I think a better solution might just be to change the row focus indicator height to `calc(100% - 1px)`

Reply:
This would cause the following change to the focus indicator, but I think that is okay, and maybe even better:

**Old:**
<img width="57" alt="Old" src="https://user-images.githubusercontent.com/8961049/224815243-a024fea2-e891-4344-88ac-1684a0b0f91f.png">

---

**New:**
<img width="57" alt="New" src="https://user-images.githubusercontent.com/8961049/224815305-eadf8f80-ddb4-43d8-8118-3ac30039a83b.png">